package.json

```diff
@@ -15,18 +15,15 @@
     "clean-all": "sf-clean all",
     "compile": "wireit",
     "docs": "sf-docs",
-    "exec-perms": "shx chmod +x lib/*.js",
     "fix-license": "eslint src test --fix --rule \"header/header: [2]\"",
     "format": "wireit",
     "link-check": "wireit",
     "lint": "wireit",
     "lint-fix": "yarn sf-lint --fix",
     "prepack": "sf-prepack",
     "prepare": "sf-install",
-    "start": "yarn build && yarn exec-perms && npm link && mcp-inspector sf-mcp-server",
```

Comment:
Was `exec-perms` needed before? I never ran it and don't start the server via `./lib/index.js` using its shebang declaration.

Reply:
I actually would like to keep this `start` command. However... it doesnt look like the perm change is needed 🤷